### ISSUE TITLE
How to disable Typescript and Jest?

### ISSUE THREAD

**Issue by KristjanESPERANTO** [GildedPleb/eslint-config-current-thing#1]
When I'm entering `...currentThing(),` I get:

```sh
/home/.../index.js
  0:0  error  Parsing error: Cannot read file '/home/.../tsconfig.json'
```

After changing it to `...currentThing({ disable: ["Typescript"] }),` nothing changes.

After creating an empty tsconfig.json file I get this:

```sh
Error: Error while loading rule 'jest/no-deprecated-functions': Unable to detect Jest version - please ensure jest package is installed, or otherwise set version explicitly
```

After changing it to `...currentThing({ disable: ["Jest", "Typescript"] }),` nothing changes.

It's probably just a small thing, but I can't figure it out at the moment.

**Comment by GildedPleb** [comment]
The config is all but predicated on Typescript and Jest. They are winning the popularity contest, and no one is happy! That is to say, I don't think you can use the config without Typescript and without Jest. After the refactor and new release, I will once again investigate if these can be turned off, but its probably far more effort than it's worth. Like the README says:

```
WARNING: disable may have unintended consequences as various configs are interdependent (To-do: This need not be so, but fixing it is very annoying).
```

**Comment by GildedPleb** [comment]
Indeed, this is doable, but too ridiculously-in-the-weeds to implement. It involves toggling various rules in every config per the plugin. And, because we can not give favoritism to any config, it also means giving the same ability to every config that cross activates / disables rules. Ie, the difficulty of adding this feature is n*n where n is the number of plugins (presently 29).

There is also probably a creative solution to the issue that can reduce this complexity, but i dont have the bandwidth to pursue it. PRs welcome! 

I will leave the issue open in case this idea gains steam.
